src/packages/auth/src/authentication/resolvers/password/resolver.ts

```diff
@@ -1,5 +1,5 @@
 import { BackendProvider, Resolver } from '@exogee/graphweaver';
-import * as argon2 from 'argon2-browser';
+import * as argon2 from 'argon2-browser/dist/argon2-bundled.min.js';
```

Comment:
This import is unusual, can you add a comment above this line to explain why we are importing the file directly?

Reply:
Good pick-up. There was a lot of context here that is now captured.